### PR TITLE
feat(backend): unread rooms query

### PR DESCRIPTION
**Pull request by Mogge**
### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: backend/src/schema/resolvers/rooms.ts [hunk 1]
```diff
@@ -25,6 +25,27 @@ export default {
...
+    UnreadRooms: async (object, params, context, resolveInfo) => {
+      const {
+        user: { id: currentUserId },
+      } = context
+      const session = context.driver.session()
+      const readTxResultPromise = session.readTransaction(async (transaction) => {
+        const unreadRoomsCypher = `
+          MATCH (:User { id: $currentUserId })-[:CHATS_IN]->(room:Room)<-[:INSIDE]-(message:Message)<-[:CREATED]-(user:User)
+          WHERE NOT user.id = $currentUserId AND NOT message.seen
+          RETURN toString(COUNT(DISTINCT room)) AS count
```
Comment (ulfgebhardt) [inline backend/src/schema/resolvers/rooms.ts:37]:
```suggestion
          MATCH (:User { id: $currentUserId })-[:CHATS_IN]->(room:Room)<-[:INSIDE]-(message:Message)<-[:CREATED]-(sender:User)
          WHERE NOT sender.id = $currentUserId AND NOT message.seen
          RETURN toString(COUNT(DISTINCT room)) AS count
```
Just ignore this if you disagree - its correct

Comment (ulfgebhardt) [reply backend/src/schema/resolvers/rooms.ts:37]: I do not understand why you make `toString`

**Review by ulfgebhardt — APPROVED** [review]
Screenshots with Seed-Data
image
image

File: backend/src/schema/resolvers/rooms.ts (hunk 1 above)
Comment (ulfgebhardt) [reply backend/src/schema/resolvers/rooms.ts]: I figured it aut <3

**Review by mahula — APPROVED** [review]

**Pull request merged**